tests/snmpsim/aviat-wtm.snmprec

```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|
```

Comment:
You must have run collect-snmp-data.php before adding your sensors, because there is no data here to test.